backend/src/v5/schemas/tickets/templates.js

```diff
@@ -55,7 +55,7 @@ const pinColSchema = Yup.lazy((val) => {
 	});
 });
 
-const blackListedChrsRegex = /^[^.,[\]]*$/;
+const blackListedChrsRegex = /^(?!\$)[^.,[\]]*$/;
```

Comment:
i think we need to block double quotes as well? otherwise that'll also cause problems..

Reply:
also `:`